independent-publisher-connectors/Edatalia sign online/apiDefinition.swagger.json

```diff
@@ -0,0 +1,625 @@
+                          "format": "int32",
+                          "description": "Widget gap in pixels to Y from text",
+                          "title": "GapY"
+                        },
+                        "rotation": {
+                          "type": "integer",
+                          "format": "int32",
+                          "description": "Widget rotation",
+                          "title": "Rotation"
+                        },
+                        "width": {
+                          "type": "integer",
+                          "format": "int32",
+                          "description": "Widget width",
+                          "title": "Width"
+                        },
+                        "height": {
+                          "type": "integer",
+                          "format": "int32",
+                          "description": "Widget height",
+                          "title": "Height"
+                        }
+                      },
+                      "description": "Widget"
+                    }
+                  },
+                  "description": "Recipient"
+                }
+              },
+              "required": [
+                "sendMethod"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/PSC/v40/DocumentSet/Evidences/{documentSetId}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Get Evidence Signed Document",
+        "description": "Gets the evidence document of the envolope process",
+        "operationId": "Get-evidence-signed-document",
+        "parameters": [
+          {
+            "name": "documentSetId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Identifier of the document generated",
+            "x-ms-summary": "Id of the document",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    },
+    "/PSC/v40/DocumentSet/Document/{documentSetId}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Get signed document (simple)",
+        "description": "Get signed document for a simple send",
+        "operationId": "Get-signed-document",
+        "parameters": [
+          {
+            "name": "documentSetId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Identifier of the document generated",
+            "x-ms-summary": "Id of the document",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    },
+    "/PSC/v40/DocumentSet/": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
empty Schema is not allowed, please define a proper schema or you can delete the empty schema.
Please fix all schemas in the swagger file